### DISCUSSION TITLE
alist如何设置跨盘复制时自动跳过已有项目？

### DISCUSSION THREAD

**Discussion by ririye** [AlistGo/alist#7353] [Q&A]
如题：跨盘复制一般时其中一个盘的挂载出问题了，删了重新挂载以后每完成的复制不能继续进行了，重新跨盘复制时要从头开始，如何能在重新复制时自动跳过已有项目呢？

**Comment by pongfcnkl** [comment]
配个rclone复制，有跳过重复文件的参数

**Reply by ririye** [reply]
具体咋搞？rclone命令不会用，求教

**Reply by pongfcnkl** [reply]
> 具体咋搞？rclone命令不会用，求教

https://rclone.org
